pkg/agent/apiserver/handlers/podinterface/handler.go

```diff
@@ -72,3 +74,14 @@ func HandleFunc(aq monitor.AgentQuerier) http.HandlerFunc {
 		}
 	}
 }
+
+var _ common.TableOutput = new(Response)
+
+func (r Response) GetTableHeader() []string {
+	return []string{"NAMESPACE", "NAME", "INTERFACE-NAME", "IP", "MAC", "PORTUUID", "OFPORT", "CONTAINER"}
```

Comment:
```suggestion
	return []string{"NAMESPACE", "NAME", "INTERFACE-NAME", "IP", "MAC", "PORT-UUID", "OF-PORT", "CONTAINER-ID"}
```

Reply:
I agree on Container-ID, I don't have a strong feeling about port-uuid and of-port

Reply:
Modified "PORTUUID", "OFPORT", "CONTAINER" to "PORT-UUID", "OF-PORT", "CONTAINER-ID".